android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -1532,10 +1536,29 @@ public static SpannableString createSpannableString(
             String text, ClickableSpan clickListener, int startIndex, int endIndex, int flags) {
         assert null != text;
         SpannableString spannableString = new SpannableString(text);
-        spannableString.setSpan(clickListener, startIndex, endIndex, flags);
+        if (startIndex >= 0 && endIndex > startIndex && endIndex < text.length()) {
+            spannableString.setSpan(clickListener, startIndex, endIndex, flags);
+        }
         return spannableString;
     }
 
+    public static Spannable createSpannableUsingHTML(
+            Context context, @StringRes int resId, View.OnClickListener onClickListener) {
+        String htmlString =
+                String.format(context.getResources().getString(resId), "<a href=\"\">", "<\\a>");
```

Comment:
I think @Pavneet-Sing pointed on that but a closing tag has to be `</a>` and not `<\a>`

Reply:
oh, may bad, will do it